src/commands/config/set.js

```diff
@@ -10,15 +11,16 @@ class ConfigSet extends BaseCommand {
     let isError = true;
     let isUserConfigUpdated = false;
     for (const flag of availableConfigs) {
+      const configProperty = camelCase(flag);
       if (this.flags[flag] !== undefined) {
         isError = false;
         this.preWarnings(flag);
         if (this.flags[flag] === '') {
           isUserConfigUpdated = await this.removeConfig(flag, isUserConfigUpdated);
           continue;
         }
-        if (await this.isOverwrite(flag)) {
-          this.userConfig[flag] = this.userConfig.sanitize(this.flags[flag]);
+        if (await this.isOverwrite(configProperty)) {
+          this.userConfig[configProperty] = this.flags[flag];
```

Comment:
Shouldn't we be calling the newly added `sanitizeFlag` here? I guess it got missed?

Reply:
yes, I missed it will update